src/OrbitCaptureClient/CaptureEventProcessorTest.cpp

```diff
@@ -502,20 +417,19 @@ GpuJob* CreateGpuJob(CaptureEvent* capture_event, uint64_t sw_queue, uint64_t hw
   return gpu_job;
 }
 
+constexpr uint64_t kTimelineKey = 17;
+constexpr const char* kTimelineString = "timeline";
```

Comment:
Same here, the internal linkage is not explicit anymore.

Reply:
Same here, very minor but still open.

Reply:
done.